documentation/index.md

```diff
@@ -51,16 +41,27 @@ fixes and constructive feedback.
 | 2     | t-run-workflow      | [9. Run Your First Workflow](/t/charmed-temporal-k8s-tutorial-run-your-first-workflow/11785)                 |
 | 2     | t-cleanup           | [10. Cleanup and Extra Info](/t/charmed-temporal-k8s-tutorial-cleanup-and-extra-info/11786)                  |
 | 1     | how-to              | [How to]()                                                                                                   |
+| 2     | h-authentication    | [Authentication](TODO)                                                                                       |
```

Comment:
Don't you have the files for these now, why are they still `TODO`?

Reply:
Once the PR is reviewed and approved, i'll be creating Discourse pages and replacing these `TODO`s with links to these pages.